packages/terra-application-layout/src/utils/utilityHelpers.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { UtilityUtils } from 'terra-application-utility';
+import UserData from '../user/_UserData';
+
+const defaultKeys = {
+  MENU: 'menu',
+  USER_INFORMATION: 'user-information',
+  CHANGE_PHOTO: 'change-photo',
+  SETTINGS: 'settings',
+  APPEARANCE: 'appearance',
+  SECURITY: 'security',
+  HELP: 'help',
+  GETTING_STARTED: 'getting-started',
+  ABOUT: 'about',
+  TERMS_OF_USE: 'terms-of-use',
+  LOG_OUT: 'log-out',
+};
+
+const menuTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.menu' }));
+const userInformationTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.userInformation' }));
+const changePhotoTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.changePhoto' }));
+const settingsTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.settings' }));
+const appearanceTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.appearance' }));
+const securityTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.security' }));
+const helpTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.help' }));
+const gettingStartedTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.gettingStarted' }));
+const aboutTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.about' }));
+const termsOfUseTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.termsOfUse' }));
+const logOutTitle = intl => (intl.formatMessage({ id: 'Terra.applicationLayout.utilityDefaults.logOut' }));
+
+const reconcileChildren = (items, additionalItems) => {
```

Comment:
Can we add a comment describing this function, the params passed in and the expected return value?